app/src/main/java/com/sdpteam/connectout/event/nearbyEvents/list/EventsAdapter.java

```diff
@@ -43,6 +43,17 @@ public EventsAdapter(@NonNull Context context, int eventListItemResource) {
         this.eventListItemResource = eventListItemResource;
     }
 
+    private static void loadOrganizerProfileImage(String organizerProfileId, ImageButton profileImageButton) {
+        // start an async job to load and display the profile image
+        new ProfileFirebaseDataSource().fetchProfile(organizerProfileId).thenAccept(profile -> {
+            final String imageUrl = profile.getProfileImageUrl();
+            if (!imageUrl.isEmpty()) {
+                Picasso.get().load(imageUrl).into(profileImageButton);
+                profileImageButton.setScaleType(ImageView.ScaleType.CENTER_CROP);
+            }
+        });
+    }
```

Comment:
I would advise against this approach, using directly here the ProfileFirebaseDataSource breaks encapsulation & MVVM paradigm. Due to the short time left, we should write an issue to fix this.